contracts/email-contracts/src/main/java/org/java/mentorship/contracts/email/dto/EmailResponse.java

```diff
@@ -0,0 +1,14 @@
+package org.java.mentorship.contracts.email.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EmailResponse {
+
+    private String message;
```

Comment:
This is fine for testing purposes
But the final version should not have a response and everything should be dictated by the response code
200 - okay
4xx - invalid request
5xx - some error occurred